services/frontend/src/components/AnalysisVisualization.vue

```diff
@@ -1,20 +1,136 @@
 <template>
-  <div>
-    <!-- create graphs here -->
-    <!-- this is a mock -->
-    <img src="mock_files/mock_analysis_graph.png"/>
+  <div v-if="analisStared" class="sector flex-column-items-centered card"> Start Analysis before starting </div>
+
+  <div v-else class="sector flex-column-items-centered card">
+
+    <div class="flex-row-items-centered" style="margin-block-end: 30px">
+
+      <table style="margin-inline-end: 30px">
+        <tr>
+          <td>Pesel </td>
+          <td>2137 </td>
+        </tr>
+        <tr>
+      <td>Imie</td>
+      <td>Daniel</td>
+        </tr>
+        <tr>
+            <td>Nazwisko</td>
+      <td>Lipniacki </td>
+        </tr>
+      </table>
+      <table>
+        <tr>
+          <td>Średni %</td>
+          <td>{{ average }}</td>
+        </tr>
+        <tr>
+            <td>Powyżej 80%</td>
+      <td>{{ counts.above }}</td>
+        </tr>
+        <tr>
+        </tr>
+      </table>
+    </div>
+    <v-chart class="chart" 
+    autoresize
+    :option="option" />
   </div>
 </template>
 
 <script>
+import { use } from "echarts/core";
+import { CanvasRenderer } from "echarts/renderers";
+import { BarChart } from "echarts/charts";
+import {
+  TitleComponent,
+  TooltipComponent,
+  LegendComponent
+} from "echarts/components";
+import VChart, { THEME_KEY } from "vue-echarts";
+
+
+use([
+  CanvasRenderer,
+  BarChart,
+  TitleComponent,
+  TooltipComponent,
+  LegendComponent
+]);
+
 export default {
   name: 'AnalysisVisualization',
   props: ['analysisData'],
-
+  components: {
+    VChart
+  },
+  provide: {
+    [THEME_KEY]: "dark"
```

Comment:
The webapp is in lightmode color palete currently, I would suggest changing it to light theme too.